tasks/buildah.yaml

```diff
@@ -113,12 +113,30 @@ spec:
     workingDir: $(workspaces.source.path)
     securityContext:
       runAsUser: 0
+  - image: registry.access.redhat.com/ubi8/python-39:1-51
+    name: convert-cyclonedx-to-purl
+    script: |
+      #!/bin/python3
+      import sys
+      import json
+
+      with open("./sbom.json") as f:
+        data = json.load(f)
+
+      purls = [component["purl"] for component in data["components"] if "purl" in component]
+
+      with open("sbom-purl", "w") as output_file:
+        for purl in purls:
+          output_file.write(f"{purl}\n")
+    workingDir: $(workspaces.source.path)
+    securityContext:
+      runAsUser: 0
```

Comment:
do we need this for all uses of the python image, or just when we were using buidlah with it (I thought it was the latter) @brunoapimentel @Michkov @ejegrova

Reply:
for all tasks which creates files, since git is cloned with user 0 (root), when you want to write into dir then you need to be root as well.

Reply:
ah that makes sense then, and I was unaware of how the git clone was occurring ... thanks for the clarification @Michkov

Reply:
marking resolved